File: isa/src/main/java/com/isa/subscriptions/dto/ResActionDTO.java
package com.isa.subscriptions.dto;

public class ResActionDTO {

    private Float price;
    private Float discount;
    private Long serviceId;

    public ResActionDTO(){}

    public ResActionDTO(Float price, Float discount, Long serviceId) {
        this.price = price;
        this.discount = discount;
        this.serviceId = serviceId;
    }

    public Float getPrice() {
        return price;
    }

    public void setPrice(Float price) {
        this.price = price;
    }

    public Float getDiscount() {
        return discount;
    }

    public void setDiscount(Float discount) {
        this.discount = discount;
    }

    public Long getServiceId() {
        return serviceId;
    }

    public void setServiceId(Long serviceId) {
        this.serviceId = serviceId;
    }
}

File: frontend/src/app/model/resActionDTO.ts
export class ResActionDTO {
    private price: number;
    private discount: number;
    private serviceId: number;


	constructor($price: number, $discount: number, $serviceId: number) {
		this.price = $price;
		this.discount = $discount;
		this.serviceId = $serviceId;
	}

}